main/ejml-simple/src/org/ejml/simple/ops/SimpleOperations_DDRM.java

```diff
@@ -68,17 +69,29 @@ public void fill( DMatrixRMaj A, /**/double value ) {
 
     @Override
     public void transpose( DMatrixRMaj input, DMatrixRMaj output ) {
-        CommonOps_DDRM.transpose(input, output);
+        if (EjmlConcurrency.useConcurrent(input)) {
+            CommonOps_MT_DDRM.transpose(input, output);
+        } else {
+            CommonOps_DDRM.transpose(input, output);
+        }
```

Comment:
Great, that users of the simple operations will simply benefit from the changes :+1: